mailpoet/lib/Config/Renderer.php

```diff
@@ -27,7 +27,7 @@ public function __construct(
       [
         'cache' => new TwigFileSystemCache($cachePath),
         'debug' => $this->debuggingEnabled,
-        'auto_reload' => true,
+        'auto_reload' => false,
```

Comment:
Hi @websupporter, Thank you for the PR.

This is just a quick confirmation.

Would running the `./do twig:generate-cache` command regenerate the twig cache files regardless of if the template was edited or not?

From the docs: 
> [auto_reload](https://symfony.com/doc/current/reference/configuration/twig.html#auto-reload)
type: boolean default: %kernel.debug%
> 
> If true, whenever a template is rendered, Symfony checks first if its source code has changed since it was compiled. If it has changed, the template is compiled again automatically.

and from the Robo command
https://github.com/mailpoet/mailpoet/blob/trunk/mailpoet/RoboFile.php#L1020-L1033

from my little understanding, it appears this would disable auto-compilation done by Twig.

Just wanted to confirm we are regenerating the cache files as it appears we are reusing the **same configuration for the Robo command**.

Reply:
Thank you for catching that @triple0t! The twig command runs at a point in time when the generated template file does not exist. But you are right: During development, this would not regenerate templates. Looking into it for a solution :thinking: